android/sandwich/src/main/kotlin/io/qonversion/sandwich/QonversionSandwich.kt

```diff
@@ -365,6 +380,48 @@ class QonversionSandwich(
         return mapOf("success" to success)
     }
 
+    private fun configurePurchaseOptions(
+        offerId: String? = null,
+        applyOffer: Boolean? = null,
```

Comment:
Let's not place default values if we always pass them
```suggestion
        offerId: String?,
        applyOffer: Boolean?,
```